### PR TITLE
Set team annotation in Chart.yaml for app alert routing

### PR DESCRIPTION
**Pull request by rossf7**

### CODE REVIEW

**Comment by njuettner** [comment]
In case we update the version, does the team also get notified?

**Review by njuettner — APPROVED** [review]

**Review by tomahawk28 — APPROVED** [review]

File: helm/cluster-api-core/Chart.yaml [hunk 1]
```diff
@@ -4,3 +4,5 @@ description: "Helm chart for cluster api core."
 home: "https://github.com/giantswarm/cluster-api-core-app"
 version: "[[ .Version ]]"
 appVersion: 0.3.19-gs1
+annotations:
+  application.giantswarm.io/team: "firecracker"
```
Comment (fiunchinho) [inline helm/cluster-api-core/Chart.yaml:8]: Wouldn't this change make alerts in Azure installations go to firecracker as well? is that expected?

Comment (rossf7) [reply helm/cluster-api-core/Chart.yaml:8]: > Wouldn't this change make alerts in Azure installations go to firecracker as well? is that expected?

@fiunchinho Yes, but there is also the owners annotation that is a bit fancier and lets us route by provider.

https://intranet.giantswarm.io/docs/dev-and-releng/app-developer-processes/routing_app_alerts/#owners-annotation

Should I change it to that?

Comment (fiunchinho) [reply helm/cluster-api-core/Chart.yaml:8]: I think so, but maybe I'm missing something. What other people think?

Comment (rossf7) [reply helm/cluster-api-core/Chart.yaml:8]: Jose, I'd like to get this merged and currently the app is only pushed to the AWS app collection.

We can easily switch to the owners annotation when it's also deployed to Azure MCs.

**Pull request merged**